from. _Error import Error


class UserNonConnessoError(Error):
    pass
